prow/jobs/busola/busola-backend/busola-backend.yaml

```diff
@@ -0,0 +1,108 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
+
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  decorate: true
+  path_alias: github.com/kyma-project/busola
+  max_concurrency: 10
+  labels:
+    preset-dind-enabled: "true"
+    preset-sa-gcr-push: "true"
+    preset-docker-push-repository-kyma: "true"
+  run_if_changed: "^backend/|^components/|^shared/|^package.json"
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/bootstrap:PR-2951
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build-generic.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-project/busola/backend"
```

Comment:
By default `build-generic.sh` runs make stage called `release`. If the desired stage has different name (e.g. `ci-release`) add it as a second argument for `build-generic.sh`

Reply:
We have release build stage in makefiles